spirv_cross/src/bindings_native.rs

```diff
@@ -1909,7 +1994,10 @@ pub mod root {
                                                 vat_override_count: usize,
                                                 p_res_overrides:
                                                     *const root::SPIRV_CROSS_NAMESPACE::MSLResourceBinding,
-                                                res_override_count: usize)
+                                                res_override_count: usize,
```

Comment:
Reminder to myself to fix the wasm build for this signature change too. We'll probably have to add the wasm build to CI to catch signature changes before it gets too far out-of-sync